operating_unit/models/operating_unit.py

```diff
@@ -42,22 +41,18 @@ class OperatingUnit(models.Model):
         ),
     ]
 
-    def name_get(self):
-        res = []
+    @api.depends("name", "code")
+    def _compute_display_name(self):
         for ou in self:
-            name = ou.name
-            if ou.code:
-                name = "[{}] {}".format(ou.code, name)
-            res.append((ou.id, name))
-        return res
+            ou.display_name = f"[{ou.code}] {ou.name}"
```

Comment:
In the previous version, we were managing the case where `ou.code` is empty -> we don't want to keep it ?

Reply:
The code fields is mandatory on the operating unit so we should not have this case